File: magic8Ball.py
import random

name = "Segun"
question = "Are You coming?"
answer = ""

random_number = random.randint(1, 9)

# Conditional Statements
if random_number == 1:
  answer = "Yes - definitely."
elif random_number == 2:
  answer = "It is decidedly so."
elif random_number == 3:
 answer = "Without a doubt."
elif random_number == 4:
  answer = "Reply hazy, try again."
elif random_number == 5:
  answer = "Ask again later."
elif random_number == 6:
  answer = "Better not tell you now."
elif random_number == 7:
  answer = "My sources say no."
elif random_number == 8:
  answer = "Outlook not so good."
elif random_number == 9:
  answer = "Very doubtful."
else:
  answer = "Error"

# If name and question string is empty
if len(question) == 0:
  print("The Magic 8-Ball cannot provide a fortune unless you ask it something.")
elif len(name) == 0:
  print("Question: " + question)
  print("Magic 8-Ball's answer: " + answer)
else:
  print(name + " asks: " + question)
  print("Magic 8-Ball's answer: " + answer)

File: magic8Ball.php
<?php

function magic8Ball(){
  echo "what is your question? \n";
  $question = readline(">> ");
  echo "\nHmmm...What an insightful question.\nI can guess what is on your mind...Your question is: $question \nHere is your answer:";
  $num_res = rand(0, 19);
  //echo "$num_res \n";

  switch ($num_res)
  {
    case 0:
    echo "It is certain. \n";
    break;
    case 1:
    echo "It is decidedly so. \n";
    break;
    case 2:
    echo "Without a doubt. \n";
    break;
    case 3:
    echo "Yes - definitely. \n";
    break;
    case 4;
    echo "You may rely on it. \n";
    break;
    case 5:
    echo "As I see it, yes. \n";
    break;
    case 6:
    echo "Most likely. \n";
    break;
    case 7:
    echo "Outlook good. \n";
    break;
    case 8:
    echo "Yes. \n";
    break;
    case 9:
    echo "Signs point to yes. \n";
    break;
    case 10:
    echo "Reply hazy, try again. \n";
    break;
    case 11:
    echo "Ask again later. \n";
    break;
    case 12:
    echo "Better not tell you now. \n";
    break;
    case 13:
    echo "Cannot predict now. \n";
    break;
    case 14:
    echo "Concentrate and ask again. \n";
    break;
    case 15:
    echo "Don't count on it. \n";
    break;
    case 16:
    echo "My reply is no. \n";
    break;
    case 17:
    echo "My sources say no. \n";
    break;
    case 18:
    echo "Outlook not so good. \n";
    break;
    case 19:
    echo "Very doubtful. \n";
    break;
  }
}

magic8Ball();
magic8Ball();
magic8Ball();

